pages/pipelines/migrate_from_jenkins.md

```diff
@@ -0,0 +1,204 @@
+# Migrate from Jenkins
+
+If you are familiar with [Jenkins](https://www.jenkins.io) and want to migrate to Buildkite, this guide is for you. Buildkite is a modern and flexible continuous integration and delivery (CI/CD) platform that provides a powerful and scalable build infrastructure for your applications.
+
+While Jenkins and Buildkite have similar goals as CI/CD platforms, their approach differs. Buildkite uses a hybrid model consisting of:
+
+- A software-as-a-service (SaaS) platform for visualization and management of CI pipelines.
+- Agents that execute jobs on your infrastructure, either on-premise or in the cloud.
```

Comment:
```suggestion
- Agents for executing jobs, hosted by you, either on-premise or in the cloud.
```

Reply:
Is the idea to more explicitly call out the "hosted by you" part?

Reply:
Yeah, I think that "Agents that execute jobs on your infrastructure" could be interpreted to mean that we (Buildkite) host the agents, but those agents run jobs on or against your (customers) infrastructure.